query.sql

```diff
@@ -0,0 +1,17 @@
+\c nc_games_test

```

Comment:
Is this your test file for SQL? If so, feel free to add it to the .gitignore file so that it's only saved locally.